include/libopencm3/stm32/lptimer.h

```diff
@@ -22,8 +22,16 @@
 #include <libopencm3/cm3/common.h>
 #include <libopencm3/stm32/memorymap.h>
 
-#if defined(STM32L0)
+#if defined(STM32F4)
+#       include <libopencm3/stm32/f4/lptimer.h>
```

Comment:
I'm still not sure I saw an f4 file?  (but yes, this is all much nicer so far)

Reply:
yeah - actually there's no lptim_base referenced in current f4 memorymap, and i got lost looking at RM (I never worked on that chip), and pushed to my branch before leaving .. - I'll update the PR after fixing that.